lib/Nekland/BaseApi/Http/ClientAdapter/GuzzleAdapter.php

```diff
@@ -26,7 +26,9 @@ class GuzzleAdapter extends AbstractHttpClient
     public function __construct(EventDispatcher $dispatcher, array $options = [], Client $client = null)
```

Comment:
Ok I just spotted the argument $client which let the user instanciate his own Client. But does this PR make the $client argument useless?